src/ngMock/angular-mocks.js

```diff
@@ -1381,15 +1381,22 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
       function handleResponse() {
         var response = wrapped.response(method, url, data, headers, wrapped.params(url));
         xhr.$$respHeaders = response[2];
-        callback(copy(response[0]), copy(response[1]), xhr.getAllResponseHeaders(),
+        var xhrStatus = copy(response[4]);
+        if (xhrStatus) {
```

Comment:
Apparantly I forgot to push things at home. I've fixed this already. I'll push it in 2 hours (+/-).